app/_components/systems/Form.tsx

```diff
@@ -0,0 +1,334 @@
+              </FormControl>
+            )}
+          />
+        </GridItem>
+
+        <GridItem w="100%">
+          <RequiresPermitField control={control} />
+        </GridItem>
+
+        <GridItem w="100%">
+          <Controller
+            name="stationFilter"
+            control={control}
+            render={({
+              field: { onChange, onBlur, value, name, ref },
+              fieldState: { error },
+            }) => (
+              <FormControl isInvalid={!!error} id="stationFilter">
+                <FormLabel>Station Filter</FormLabel>
+                <Select<SelectGroup, true, GroupBase<SelectGroup>>
+                  id="stationFilter-field"
+                  instanceId="stationFilter-field"
+                  name={name}
+                  ref={ref}
+                  onBlur={onBlur}
+                  value={value}
+                  options={[
+                    { label: 'Has Stations', value: 'hasStations' },
+                    { label: 'Has Planetary', value: 'hasPlanetary' },
+                    { label: 'Has Orbital', value: 'hasOrbital' },
+                    { label: 'Has No Stations', value: 'hasNoStations' },
+                  ]}
+                  chakraStyles={chakraSelectStyles}
+                />
+                <FormErrorMessage>{error && error.message}</FormErrorMessage>
+              </FormControl>
+            )}
+          />
+        </GridItem>
+
+        <GridItem w="100%">
+          <PowersField control={control} />
+        </GridItem>
+
+        <GridItem w="100%">
+          <PowerEffectsField control={control} />
+        </GridItem>
+
+        <GridItem w="100%" colSpan={{ base: 1, md: 1, lg: 2 }}>
+          <FormControl
+            isInvalid={
+              !!(errors.referenceSystem && errors.referenceSystem.message)
+            }
+          >
+            <FormLabel>Reference System</FormLabel>
+            <Input
+              variant="outline"
+              placeholder="Enter a system..."
+              borderColor={selectColor(isDark, 'border')}
+              _hover={{
+                borderColor: selectColor(isDark, 'border'),
+              }}
+              {...register('presenceType', {
+                required: true,
+                pattern: /^[\w'-]+(?:\s[\w'-]+)*$/,
+                maxLength: 40,
+              })}
+            />
+            <FormErrorMessage>
+              {errors.referenceSystem && errors.referenceSystem.message}
+            </FormErrorMessage>
+          </FormControl>
+        </GridItem>
+
+        <GridItem w="100%">
+          <Controller
+            name="securities"
+            control={control}
+            render={({
+              field: { onChange, onBlur, value, name, ref },
+              fieldState: { error },
+            }) => (
+              <FormControl isInvalid={!!error} id="securities">
+                <FormLabel>Securities</FormLabel>
```

Comment:
Me being pedantic about the English language here but I think this should be 'security level'